evaluation/mint/requirements.txt

```diff
@@ -0,0 +1,32 @@
+pre-commit
+openai
+datasets
+backoff
+charset-normalizer==3.1.0
+# Alfworld
+pandas==1.4.4
+opencv-python
+networkx
+tqdm
+vocab
+revtok
+Click
+ai2thor==2.1.0
+transformers
+tokenizers
+scipy==1.10.1
+ipython
+matplotlib
+cython
+nltk
+gym==0.15.4
+pipreqs
+pyyaml
+pytz
+visdom
+sympy
+pycocotools
+seaborn
+google-generativeai
+python-dateutil
+statsmodels
```

Comment:
I'm not quite clear about how modules in that section are used in the code, can you explain a little bit? Because I need to install them in the `DockerSSHBox` before the evaluation, but simply moving them to `pyproject.toml` seems not to work.

Reply:
Aha, never mind! I didn't read your code logic yet - please ignore this comment then.